treeherder/etl/perf.py

```diff
@@ -173,6 +146,44 @@ def load_perf_artifacts(project_name, reference_data, job_data, datum):
                 generate_alerts.apply_async(args=[signature.id],
                                             routing_key='generate_perf_alerts')
 
+        for property in subtest_properties:
```

Comment:
As above, let's rename `property` to `subtest_metadata`. I think that reads a little easier.